_layouts/platforms.html

```diff
@@ -30,7 +30,7 @@ <h2 class="subtitle">{{ page.subtitle }}</h2>
                 </figure>
                 {% endif %}
                 <h1 class="projectslist__listing__title">{{platform.title}}</h1>
-                <p>{{platform.subtitle}}</p>
+                <p class="projectslist__listing__subtitle">{{platform.subtitle}}</p>
```

Comment:
We can use the class provided by Bootstrap / Bootstrap italia here.

Reply:
ah ok, u mean something like this instead ? In this way I could delete the code that I added on the main.scss file.
`<p class="text-dark">{{platform.subtitle}}</p>`

Reply:
Exactly, `text-secondary` should do.